lib/diaspora_federation/entities/like.rb

```diff
@@ -1,10 +1,30 @@
 module DiasporaFederation
   module Entities
+    # this entity represents a like to some kind of post (e.g. status message)
+    #
+    # @see Validators::LikeValidator
     class Like < Entity
+      # @!attribute [r] positive
+      #   If true set a like, if false, remove it
```

Comment:
i thought positive=false is a dislike (which is not implemented, but possible by the protocol)? and removing a like is done by a retraction?